stdlib/concurrent/futures/process.pyi

```diff
@@ -1,28 +1,164 @@
 import sys
+import threading
+import weakref
+from collections.abc import Generator, Iterable, Mapping, MutableMapping, MutableSequence
+from concurrent.futures._base import Executor, Future
+from multiprocessing.connection import Connection
+from multiprocessing.context import BaseContext, Process
+from multiprocessing.queues import Queue, SimpleQueue
+from types import TracebackType
 from typing import Any, Callable, Tuple
 
-from ._base import Executor
+_threads_wakeups: MutableMapping[Any, Any]
+_global_shutdown: bool
 
-EXTRA_QUEUED_CALLS: Any
+class _ThreadWakeup:
+    _closed: bool
+    _reader: Connection
+    _writer: Connection
+    def __init__(self) -> None: ...
+    def close(self) -> None: ...
+    def wakeup(self) -> None: ...
+    def clear(self) -> None: ...
+
+def _python_exit() -> None: ...
+
+EXTRA_QUEUED_CALLS: int
+
+_MAX_WINDOWS_WORKERS: int
+
+class _RemoteTraceback(Exception):
+    tb: str
+    def __init__(self, tb: TracebackType) -> None: ...
+    def __str__(self) -> str: ...
+
+class _ExceptionWithTraceback:
+    exc: BaseException
+    tb: TracebackType
+    def __init__(self, exc: BaseException, tb: TracebackType) -> None: ...
+    def __reduce__(self) -> str | Tuple[Any, ...]: ...
+
+def _rebuild_exc(exc: Exception, tb: str) -> Exception: ...
+
+class _WorkItem(object):
```

Comment:
Nit: Please don't derive from `object` as that is implicit in Python 3. (Same for other classes below.)

Reply:
Ah so in these cases, simply changing it from `class _WorkItem(object)` to `class _WorkItem` should work correctly?
I was basing this off what the code used in the Python which explicitly defined those classes as inheriting from `object`.

Reply:
`class _WorkItem` is correct. In Python 2 there was a difference between deriving from `object` or not deriving, this is often a relict of that. Nowadays it's just a stylistic issue.